README.md

```diff
@@ -526,10 +527,49 @@ Supported options for INSTAGRAM_STORIES:
 | **SHARE_STICKER_IMAGE** | stickerImage   |
 | **SHARE_BACKGROUND_AND_STICKER_IMAGE** | backgroundImage, stickerImage   |
 
+### Static Values for Facebook Stories
+These can be assessed using Share.Social property
+For eg.
+```javascript
+import Share from 'react-native-share';
+
+const shareOptions = {
+    method: Share.FacebookStories.SHARE_BACKGROUND_AND_STICKER_IMAGE,
+    backgroundImage: 'http://urlto.png',
+    stickerImage: 'data:image/png;base64,<imageInBase64>', //or you can use "data:" link
+    backgroundBottomColor: '#fefefe',
+    backgroundTopColor: '#906df4',
+    attributionURL: 'http://deep-link-to-app', //in beta
+    appId: '219376304', //facebook appId
```

Comment:
This is the appId on Facebook Pages?

Reply:
Yes that's right, it's required from Facebook.
With Instagram we doesn't need, but with Facebook they required.